src/gui/previewselectdialog.cpp

```diff
@@ -96,7 +96,6 @@ PreviewSelectDialog::PreviewSelectDialog(QWidget *parent, BitTorrent::TorrentHan
         }
     }
 
-    connect(this, SIGNAL(readyToPreviewFile(QString)), parent, SLOT(previewFile(QString)));
```

Comment:
The removal of this line causes the previewFile not working.

Reply:
It is moved to another location... and it still works correctly for me.

Reply:
Sorry, it's not working for me. Tested on ArchLinux and Ubuntu-19.04

Reply:
Please file an issue, we need more details.

Reply:
Works for me.